signer/tests/transaction.rs

```diff
@@ -106,6 +106,10 @@ impl Transaction {
         self
     }
 
+    pub fn get_memo(&self) -> [u8; MEMO_BYTES] {
```

Comment:
since there is a setter, but we access `memo`, here's a getter so `memo` can remain privatie